src/foam/nanos/auth/Country.js

```diff
@@ -16,12 +16,21 @@ foam.CLASS({
     {
       class: 'String',
       name: 'code',
-      documentation: 'Country code.'
+      documentation: 'ISO 3166-1 alpha-2 Country codes.'
+    },
+    {
+      class: 'String',
+      name: 'ISO_3166_1',
```

Comment:
This is going to make for odd method names. Can we rename to iso31661Code or iso3166Code?

Reply:
Rename to iso31661Code